lib/improver/tests/blending/weighted_blend/test_conform_metadata.py

```diff
@@ -73,6 +73,11 @@ def setUp(self):
         cube_orig.transpose([1, 0, 2, 3])
         cube_orig = cube_orig[0]
```

Comment:
I know this isn't new in this PR, but can you add this comment to help everyone know what's going on:
```suggestion
        cube_orig = cube_orig[0]  # Removing realization coordinate.
```